doc/release/2.11.0.rst

```diff
@@ -0,0 +1,586 @@
+
+
+
+.. _enterprise:
+
+Enterprise
+----------
+
+
+.. _security_enhancements:
+
+Security enhancements
+~~~~~~~~~~~~~~~~~~~~~
+
+
+.. _encrypted_ssl_tls_keys:
+
+Encrypted SSL/TLS keys
+**********************
+
+Tarantool Enterprise now supports `encrypted <https://www.tarantool.io/en/enterprise_doc/security/#traffic-encryption>`_ SSL/TLS private key files protected with a password.
+Given that most certificate authorities generate encrypted keys, this feature simplifies the maintenance of Tarantool instances.
+
+A password can be provided using either the new ``ssl_password`` URI parameter or in a text file specified using ``ssl_password_file``, for example:
+
+.. code-block:: lua
+
+    box.cfg{ listen = {
+        uri = 'localhost:3301',
+        params = {
+            transport = 'ssl',
+            ssl_key_file = '/path_to_key_file',
+            ssl_cert_file = '/path_to_cert_file',
+            ssl_ciphers = 'HIGH:!aNULL',
+            ssl_password = 'topsecret'
+        }
+    }}
+
+
+.. _security_enforcement_options:
+
+Security enforcement options
+****************************
+
+With v2.11, Tarantool Enterprise includes new security enforcement options.
+These options provide the ability to improve security by flexible customization of a password policy.
+A password policy enables you to enforce the use of strong passwords, set up a maximum password age, and so on.
+For example, the ``password_min_length`` configuration option specifies the minimum number of characters for a password:
+
+.. code-block:: lua
+
+    box.cfg{ password_min_length = 10 }
+
+To specify the maximum period of time (in days) a user can use the same password, you can use the ``password_lifetime_days`` option, which uses the system clock under the hood:
+
+.. code-block:: lua
+
+    box.cfg{ password_lifetime_days = 365 }
+
+When this period ends, a user gets the 'Password expired' error on a login attempt.
+Note that by default, new options are not specified.
+You can learn more about all the available options from the
+`Authentication restrictions <https://www.tarantool.io/en/enterprise_doc/security/#authentication-restrictions>`_ and
+`Password policy <https://www.tarantool.io/en/enterprise_doc/security/#password-policy>`_ sections.
+
+
+.. _pap_sha_256:
+
+PAP-SHA256 authentication method
+********************************
+
+By default, Tarantool uses the CHAP protocol to authenticate users and applies SHA-1 hashing to passwords.
+In this case, password hashes are stored in the ``_user`` space unsalted.
+If an attacker gains access to the database, they may crack a password, for example, using a rainbow table.
+
+With Tarantool Enterprise, you can enable PAP authentication with the SHA256 hashing algorithm.
```

Comment:
Same as above: can we express the essence with less words? I don't think that we need a second mentioning of cracking and rainbow tables.